htmresearch/algorithms/faulty_temporal_memory.py

```diff
@@ -58,6 +49,7 @@ def killCells(self, percent = 0.05):
     time you call this method a permutation list is set up. Calls change the
     number of cells considered dead.
     """
+
```

Comment:
If so, it doesn't affect the performance and results in the same figures as without.